services/ui-src/src/utils/other/mask.tsx

```diff
@@ -0,0 +1,57 @@
+/*
+ * Custom Masks Type Guard
+ * Add any future custom masks here!
+ */
+const customMasks = ["comma-seperated"] as const;
+export type CustomMasks = typeof customMasks[number];
+export const isCustomMask = (x: any): x is CustomMasks =>
+  customMasks.includes(x);
+
+/**
+ * Checks if the provided string is contains numbers to mask
+ * @param {String} value
+ * @returns {String}
+ */
+function isCustomValueMaskable(value: string): boolean {
+  if (value && typeof value === "string") {
+    const hasDigits = value.match(/\d/);
+    if (hasDigits) {
+      return true;
+    }
+  }
+  return false;
+}
+
+/**
+ * Converts a number-like string to a comma seperated value
+ * @param {String} value
+ * @returns {String}
+ */
+function toCommaSeperated(value: string): string {
+  // Convert String to a float to begin operation
+  const valueAsFloat = parseFloat(value);
+
+  // Slide any extra decimals down to 2
+  const fixedDecimal = valueAsFloat.toFixed(2);
+
+  // Clean up the float value and add in commas to delineate thousands if needed
+  const cleanedValue = Number(fixedDecimal).toLocaleString("en", {
+    minimumFractionDigits: 2,
+  });
+
+  return `${cleanedValue}`;
+}
```

Comment:
Typo in function name. We might also need to make sure any commas in the value string are removed prior to this, in case users have entered some, because `parseFloat()` will not work correctly if there's anything but numbers and decimals. Also we only need to call `toFixed()` if there is a decimal in the number to start with I think.

```suggestion
const convertToCommaSeparatedString = (value: string): string => {
  const valueAsFloat = parseFloat(value);
  const fixedDecimal = valueAsFloat.toFixed(2);
  // clean up float value; add commas if needed
  const cleanedValue = Number(fixedDecimal).toLocaleString("en", {
    minimumFractionDigits: 2,
  });
  return cleanedValue.toString();
}
```

Reply:
True! Cleaned up and tested 💪